src/uct/ib/rc/accel/rc_mlx5_common.c

```diff
@@ -440,6 +442,7 @@ ucs_status_t uct_rc_mlx5_iface_common_init(uct_rc_mlx5_iface_common_t *iface,
                                   "rc_mlx5_atomic_desc");
     if (status != UCS_OK) {
         UCS_STATS_NODE_FREE(iface->stats);
+        uct_mlx_iface_dm_cleanup(iface);
```

Comment:
seems better use goto, because you also need to clean it if `UCS_STATS_NODE_ALLOC` fails as well.

Reply:
fixed